src/libs/actions/Task.ts

```diff
@@ -295,14 +304,37 @@ function createTaskAndNavigate(
     Report.notifyNewAction(parentReportID, currentUserAccountID);
 }
 
+/**
+ * @returns the object to update `report.hasOutstandingChildTask`
+ */
+function getOutstandingChildTask(taskReport: OnyxEntry<OnyxTypes.Report>) {
+    const parentReportActions = allReportActions?.[`${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${taskReport?.parentReportID}`] ?? {};
+    return Object.values(parentReportActions).some((reportAction) => {
+        if (String(reportAction.childReportID) === String(taskReport?.reportID)) {
+            return false;
+        }
+
+        if (
+            reportAction.childType === CONST.REPORT.TYPE.TASK &&
+            reportAction?.childStateNum === CONST.REPORT.STATE_NUM.OPEN &&
+            reportAction?.childStatusNum === CONST.REPORT.STATUS_NUM.OPEN &&
+            ReportActionsUtils.getReportActionMessage(reportAction)?.isDeletedParentAction
```

Comment:
This line causes the green dot to flicker when we check any Task. Instead, it should check for negation of `isDeletedParentAction`. It looks like a bad merge. More context can be found in this Proposal: https://github.com/Expensify/App/issues/55685#issuecomment-2611323625